### PR TITLE
Handle "NO ACTION TAKEN" history item in LA Metro scraper

### PR DESCRIPTION
**Pull request by hancush**
This PR adds a key for "NO ACTION TAKEN" to the `ACTION_CLASSIFICATION` dict.

### CODE REVIEW

File: lametro/bills.py [hunk 1]
```diff
@@ -319,7 +319,8 @@ def scrape(self, window=28, matter_ids=None) :
                          'CARRIED OVER' : 'deferral',
                          'RECEIVED' : 'receipt',
                          'REFERRED' : 'referral-committee',
-                         'FORWARDED DUE TO ABSENCES AND CONFLICTS' : 'committee-passage'}
+                         'FORWARDED DUE TO ABSENCES AND CONFLICTS' : 'committee-passage',
+                         'NO ACTION TAKEN': 'no-action-taken'}
```
Comment (fgregg) [inline lametro/bills.py]: the possible values that can appear are part of controlled vocabulary: https://github.com/opencivicdata/python-opencivicdata/blob/dfbb926080b0698b4885cd5f1a08637117416021/opencivicdata/common.py#L113

I don't think it's worth adding a new type to that controlled vocabular, so I would set the value to `None`

Comment (hancush) [reply lametro/bills.py]: okey doke, reverted back to my original commit, which does exactly that.

**Comment by hancush** [comment]
fyi @fgregg, metro followed up on the email thread and said "filing" made sense to them, so i updated the classification appropriately.

**Comment by fgregg** [comment]
okay, still approve

**Pull request merged**